types/setting.go

```diff
@@ -255,6 +261,34 @@ var (
 		Default:     "300",
 	}
 
+	SettingDefinitionFailedBackupAutoDeleteion = SettingDefinition{
+		DisplayName: "Failed Backup Auto-Deletion",
+		Description: "This setting allows Longhorn to delete the backup resource that was failed automatically.\n\n" +
+			"WARNING: \n\n" +
+			"  - The failed backup will be removed in **BackupstorePollInterval** (default: 5 mintues) and you can only check the backup failed from event log.\n" +
+			"  - Keeping lots of the failed backups might affect the performance for backup volumes, upgrading or uninstallating longhorn system.\n\n" +
+			"Note: \n\n" +
+			"  - Setting 0 to disable **Backupstore Poll Interval** also means to disable failed Backup auto-deletion.\n",
+		Category: SettingCategoryBackup,
+		Type:     SettingTypeBool,
+		Required: true,
+		ReadOnly: false,
+		Default:  "true",
+	}
+
+	SettingDefinitionFailedBackupTTL = SettingDefinition{
+		DisplayName: "Failed Backup Time to Live",
+		Description: "In seconds. This setting determines how long Longhorn will keep the backup resource that was failed.\n\n" +
+			"Note: \n\n" +
+			"  - Setting 0 to disable **Backupstore Poll Interval** also means to disable failed backup time to live.\n" +
+			"  - Disabling **Failed Backup Auto-Deletion** would disable failed backup time to live as well.\n",
```

Comment:
Suggest combining `"In seconds. This setting determines how long Longhorn will keep the backup resource that was failed.` and `Disabling **Failed Backup Auto-Deletion** would disable failed backup time to live as well.` and putting it into description.

Reply:
Combined.